crates/biome_css_analyze/src/lint/nursery/no_unknown_property.rs

```diff
@@ -54,7 +54,7 @@ declare_rule! {
     /// ```
     ///
     pub NoUnknownProperty {
-        version: "1.8.0",
+        version: "next",
```

Comment:
I don't think we should update the version. The rule is introduced in v1.8.0, a bug fix like this shouldn't refresh its birthday.

Reply:
@Sec-ant 
Ah, thanks